examples/gp/linear_verif/gpmsa_linear_verif.C

```diff
@@ -0,0 +1,345 @@
+// Bayes linear verification problem due to Brian Williams
```

Comment:
Can you put the QUESO licence before this?  You can copy it from one of the other examples.